### PR TITLE
Added Recommendation Cards API endpoint

### PR DESCRIPTION
**Pull request by ahmed605**

### CODE REVIEW

**Review by yoshiask — CHANGES_REQUESTED** [review]

File: MicrosoftStore/Models/Recommendations.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MicrosoftStore.Models
+{
+    public class Recommendations
```
[the remark this answers is not in the document]
Comment (ahmed605) [reply MicrosoftStore/Models/Recommendations.cs]: Fixed!

File: MicrosoftStore/Enums/ImageType.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MicrosoftStore.Enums
+{
+    public enum ImageType
+    {
+        BoxArt,
```
[the remark this answers is not in the document]
Comment (ahmed605) [reply MicrosoftStore/Enums/ImageType.cs:14]: I was adding the attributes to the model properties directly but I moved them to the enum now

File: MicrosoftStore/IStorefrontApi.cs [hunk 3]
```diff
@@ -19,5 +20,11 @@ public interface IStorefrontApi
         /// </summary>
         [Get("/products/{productId}")]
         Task<ResponseItem> GetProduct(string productId, string market, string locale);
+
+        /// <summary>
+        /// Gets trending recommendation cards of home page
+        /// </summary>
+        [Get("/recommendations/collections/Collection/TrendingHomeColl1?market={market}&locale={locale}&deviceFamily=Windows.Desktop&appVersion={appversion}&pageSize={pageSize}&cardsEnabled=true")]
+        Task<Recommendations> GetHomeRecommendations(string market = "US", string locale = "en-US", string appversion = "11905.1001.0.0", int pageSize = 15);
```
[the remark this answers is not in the document]
Comment (ahmed605) [reply MicrosoftStore/IStorefrontApi.cs]: Nope, just making it easy to use the endpoint without using many arguments

File: MicrosoftStore/Enums/ImageType.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (yoshiask) [reply MicrosoftStore/Enums/ImageType.cs:14]: I mean on the enum values. It doesn't need to have specified strings for each enum value, or are they all deserialized properly?

**Review by yoshiask — CHANGES_REQUESTED** [review]

File: MicrosoftStore/Enums/ImageType.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (ahmed605) [reply MicrosoftStore/Enums/ImageType.cs:14]: it needs this in case if vales names don't match their names in JSON only

**Review by yoshiask — APPROVED** [review]

**Pull request merged**